colossus-metrics/src/main/resources/reference.conf

```diff
@@ -0,0 +1,20 @@
+colossus {
+  metrics {
+    collect-system-metrics: true
+    metric-intervals: ["1 second", "1 minute"]
+    metric-address: "/"
```

Comment:
probably better names for these parameters are "aggregation-intervals" and "root-address"....or something else, I'm open to ideas, but prefixing them with "metric" seems redundant.